lib/Interfaces/ChannelInterface.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
I'd argue this should be in VideoManagerMeta, or at least the part that is generic between VM6 and VM Pro